### DISCUSSION TITLE
Meeting cancellation due to sickness

### DISCUSSION THREAD

**Discussion by sarafarahabadi** [sovelluskontti/project-semantic-search#62] [General]
Hello Turo, 

I got bad flu over the weekend and I can not talk since I have a very bad sore throat. Is it possible to cancel or postpone today's meeting? 

Since the basic code and inserting the data to the manticore table works now. I am working on documentation and after it doing the full-text search on Flask and React. I will keep the GitHub board up-to-date and move on with the goal. 

Best regards,
Sara

**Comment by nyluntu** [comment]
Hi.

That's ok for me. Let's see if we need any meetings during the week. We can meet again on the next monday if you don't have any big issues.

**Reply by sarafarahabadi** [reply]
Thank you very much. I will contact you if I have any big issues.

**Reply by nyluntu** [reply]
I put the work certificate into shared folder if you can check it. Tell me if any changes is required. Check the dates that the start and end dates are right.

**Reply by sarafarahabadi** [reply]
Thank you so much. The dates are correct. I will upload it for the university and let you know if any changes are required.
